docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -77,17 +77,18 @@ To build documentation locally, ensure you have a working :ref:`development envi
 To work with documentation on your local machine, you need to have python-3.5 or greater and the
 following packages installed:
 
-- gcc
-- jinja2
-- libyaml
-- Pygments >= 2.4.0
-- pyparsing
-- PyYAML
-- rstcheck
-- six
-- sphinx
-- sphinx-notfound-page
-- straight.plugin
+    - ``gcc``
+    - ``jinja2``
```

Comment:
Wouldn't it be easier to point to requirements/constraints files that already have this info?